projects/ossf-scorecard/yaml_fuzzer.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package policy
+
+func FuzzParseFromYAML(data []byte) int {
```

Comment:
Thanks, I would rather have the fuzzing code within the scorecard. Is there a specific reason to keep idt outside?

Reply:
If we change the API now someone in oss-fuzz would have to maintain this, with go 1.18 support fuzzing would be have been much easier compared to go-fuzz

Reply:
No problem. I will make a PR upstream with a Go 1.18 fuzzer shorty.

Reply:
Go 1.18 fuzzer added here: https://github.com/ossf/scorecard/pull/1631